src/types/hbs.rs

```diff
@@ -152,7 +152,8 @@ impl HbsClient {
 
         let email = match config {
             hpos_config_core::Config::V1 { settings, .. }
-            | hpos_config_core::Config::V2 { settings, .. } => settings.admin.email,
+            | hpos_config_core::Config::V2 { settings, .. }
+            | hpos_config_core::Config::V3 { settings, .. } => settings.admin.email,
```

Comment:
can we just have a function that returns the email so that a new line does not need to be added every time here?
Eg:
```
let email = config.email();
```

Reply:
Done (on multiple repos) and hashes updated.